views/default/plugins/upload_users/settings.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+// Custom message in email notification
+echo '<p><label>' . elgg_echo('upload_users:notification:custom'). '</label> ' . elgg_view('input/longtext', array('name' => 'params[custom_message]', 'value' => $vars['entity']->custom_message)) . '</p>';
```

Comment:
One tag per line. It's more readable if you use inline html and then echo php strings:

```php
?>
<div>
   <label><?php echo 'label' ?></label>
   <?php echo $input ?>
</div>
```